package.json

```diff
@@ -18,23 +21,24 @@
   "author": "Richard Rodger (http://richardrodger.com)",
   "license": "MIT",
   "dependencies": {
-    "fivebeans": "~1.3.1",
-    "lodash": "^3.10.1",
+    "fivebeans": "~1.5.0",
+    "lodash": "4.15.0",
     "nid": "~0.3.2"
   },
-  "peerDependencies" : {
-    "seneca": "0.7.x"
-  },
   "devDependencies": {
-    "eslint-config-standard": "^4.4.0",
-    "eslint-plugin-standard": "^1.3.1",
-    "lab": "^6.2.0",
+    "eslint-config-standard": "^6.0.0",
```

Comment:
Don't need this dep, think we should have eslint-plugin-standard instead https://github.com/senecajs/seneca/blob/master/package.json#L105